From 4e91f30f5b0b46a6d7f8f7adfb9e267bcb2e7450 Mon Sep 17 00:00:00 2001
From: Pan Jiafei <Jiafei.Pan@nxp.com>
Date: Fri, 15 Jul 2016 10:14:51 +0800
Subject: [PATCH 138/508] fmd: change HC conf and err fq to a higher priority

To avoid timeout for waiting host command confirmation when
there is another heavy load enqueued to the same Core.

Signed-off-by: Pan Jiafei <Jiafei.Pan@nxp.com>
[Original patch taken from SDK-V2.0-1703]
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../sdk_fman/src/wrapper/lnxwrp_fm_port.c          |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c
index fc90bfb..92cd92d 100644
--- a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c
+++ b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c
@@ -1195,7 +1195,7 @@ static t_Error InitFmPcdDev(t_LnxWrpFmDev *p_LnxWrpFmDev)
 			FqAlloc(p_LnxWrpFmDev,
 				0,
 				QMAN_FQ_FLAG_NO_ENQUEUE,
-				p_LnxWrpFmDev->hcCh, 7);
+				p_LnxWrpFmDev->hcCh, 1);
 		if (!p_LnxWrpFmDev->hc_tx_conf_fq)
 			RETURN_ERROR(MAJOR, E_NULL_POINTER,
 				     ("Frame queue allocation failed..."));
@@ -1204,7 +1204,7 @@ static t_Error InitFmPcdDev(t_LnxWrpFmDev *p_LnxWrpFmDev)
 			FqAlloc(p_LnxWrpFmDev,
 				0,
 				QMAN_FQ_FLAG_NO_ENQUEUE,
-				p_LnxWrpFmDev->hcCh, 7);
+				p_LnxWrpFmDev->hcCh, 2);
 		if (!p_LnxWrpFmDev->hc_tx_err_fq)
 			RETURN_ERROR(MAJOR, E_NULL_POINTER,
 				     ("Frame queue allocation failed..."));
-- 
1.7.5.4

